specification/schemaregistry/SchemaRegistry/tspconfig.yaml

```diff
@@ -0,0 +1,39 @@
+parameters:  
+  "service-dir":
+    default: "sdk/schemaregistry"
+  "dependencies":
+    "additionalDirectories": []
+    default: ""
+emit:
+  - "@azure-tools/typespec-autorest"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-core/all"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/../data-plane/SchemaRegistry/preview/2023-07-01"
+    output-file: "SchemaRegistry.json"
```

Comment:
Just curious, any reason we're not defaulting to openapi.json?

Reply:
Since I suspect this needs to replace the existing GA swagger, the rename is fine, let's just match the casing for the existing swagger file name.

Reply:
I can make it openapi.json, but just didn't b/c I haven't seen that elsewhere. As a follow-up, _should_ this replace the existing swagger?

Reply:
This should replace the existing file, so just name it the same as the original

Reply:
Naming it the same as the original. Given that the original swagger is in the Microsoft.EventHub folder and Johan mentioned that we want it to be in the Azure.SchemaRegistry folder, I don't think it will overwrite it though but will at least follow convention.